test/rest/SearchEndpointTest.ts

```diff
@@ -23,7 +23,7 @@ module Coveo {
       var ep: SearchEndpoint = Coveo.SearchEndpoint.endpoints['default'];
```

Comment:
I don't see any explicit test that check that workGroup gets transformed in organizationId.

It's all implicit, because the setup create workgroup=, but the test check for organizationId=. 

I would very much prefer an explicit test with an explicit title that check for this. 

That means that if we break this in the future, the UT will much more helpful to see what's wrong.